conans/test/functional/generators/cmake_multi_test.py

```diff
@@ -272,6 +272,48 @@ def cmake_multi_test(self):
             self.assertIn("Hello Release Hello0", client.out)
 
 
+class CMakeMultiSystemLibsTest(unittest.TestCase):
+
+    def system_libs_test(self):
+        mylib = textwrap.dedent("""
+            import os
+            from conans import ConanFile
+
+            class MyLib(ConanFile):
+
+                def package_info(self):
+                    self.cpp_info.debug.system_libs = ["sys1d"]
+                    self.cpp_info.release.system_libs = ["sys1"]
+                    self.cpp_info.libs = ["lib1"]
```

Comment:
Try to enter also `libs.release` and `libs.debug` to assert that the order in the CONAN_LIBS_DEBUG etc is correct